File: py/ege25.78v3.py
def isPrime(x):
    if x < 2:
        return False
    
    j = 2
    while j * j <= x:
        if x % j == 0:
            return False
        j += 2 if j > 2 else 1

    return True

n = 0
N = 200000
for i in range(2, N + 1):
    if isPrime(i):
        n += 1

print(n)

File: c/ege25.78v3.c
#include <stdio.h>

int isPrime(int x) {
    if(x < 2) return 0;    

    int j = 2;
    while(j * j <= x) {
        if(x % j == 0) {
            return 0;
        }
        j += j > 2 ? 2 : 1;  
    }

    return 1;
}

int main() {
    int N = 200000;
    int n = 0;

    for(int i = 2; i <= N; i++) {
        if(isPrime(i))
            n++;
    } 

    printf("%d\n", n);
}

